helm/values.yaml

```
@@ -1,7 +1,7 @@
 # configure Connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v2.1.2
+  image: securesystemsengineering/connaisseur:v2.2.0
   helmHookImage: securesystemsengineering/connaisseur:helm-hook-v1.1
   imagePullPolicy: Always
```

```suggestion
  imagePullPolicy: IfNotPresent
```
Dow we actually need `Allways`?